clients/ops/admin-ui/src/features/datastore-connections/add-connection/sass/ConnectorParameters.tsx

```diff
@@ -0,0 +1,234 @@
+import {
+  Button,
+  ButtonGroup,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+  Textarea,
+  Tooltip,
+  VStack,
+} from "@fidesui/react";
+import { CircleHelpIcon } from "common/Icon";
+import { capitalize } from "common/utils";
+import {
+  ConnectionOption,
+  ConnectionTypeSecretSchemaReponse,
+} from "connection-type/types";
+import { useLazyGetDatastoreConnectionStatusQuery } from "datastore-connections/datastore-connection.slice";
+import { Field, Form, Formik } from "formik";
+import React from "react";
+
+import { CustomFields } from "../types";
+
+const defaultValues: CustomFields = {
+  name: "",
+  description: "",
+  connectionIdentifier: "",
+};
+
+type ConnectorParametersProps = {
+  connectionOption: ConnectionOption;
+  data: ConnectionTypeSecretSchemaReponse;
+  onTestConnectionClick: (value: any) => void;
+};
+
+export const ConnectorParameters: React.FC<ConnectorParametersProps> = ({
+  connectionOption,
+  data,
+  onTestConnectionClick,
+}) => {
+  const [trigger, result] = useLazyGetDatastoreConnectionStatusQuery();
+
+  const validateField = (label: string, value: string) => {
+    let error;
+    if (!value) {
+      error = `${label} is required`;
+    }
+    return error;
+  };
+
+  const getFormLabel = (id: string, value: string): JSX.Element => (
+    <FormLabel
+      color="gray.900"
+      fontSize="14px"
+      fontWeight="semibold"
+      htmlFor={id}
+      minWidth="141px"
+    >
+      {value}
+    </FormLabel>
+  );
+
+  const getFormField = (key: string, item: { title: string }): JSX.Element => (
+    <Field
+      id={key}
+      name={key}
+      key={key}
+      validate={
+        data.required.includes(key)
+          ? (value: string) => validateField(item.title, value)
+          : false
+      }
+    >
+      {({ field, form }: { field: any; form: any }) => (
+        <FormControl
+          display="flex"
+          isRequired={data.required.includes(key)}
+          isInvalid={form.errors[key] && form.touched[key]}
+        >
+          {getFormLabel(key, item.title)}
+          <VStack align="flex-start" w="inherit">
+            <Input {...field} color="gray.700" size="sm" />
+            <FormErrorMessage>{form.errors[key]}</FormErrorMessage>
+          </VStack>
+          <CircleHelpIcon marginLeft="8px" />
+        </FormControl>
+      )}
+    </Field>
+  );
+
+  const getInitialValues = () => {
+    Object.entries(data.properties).forEach((key) => {
+      defaultValues[key[0]] = "";
+    });
+    return defaultValues;
+  };
+
+  const handleSubmit = (values: any, actions: any) => {
+    actions.setSubmitting(false);
+  };
+
+  const handleTestConnectionClick = () => {
+    // TODO: Replace the connection key value with the
+    // actual connection key value from the handleSubmit function
+    // trigger("ci_create_test_data_datastore_connection_disabled").then(
+    trigger("app_postgres_db").then((response) => {
```

Comment:
This looks like it ought to be for the key specified in the `Connection Identifier` box?

Reply:
Hard-coded value has been replaced.